pym/bob/generators/QtCreatorGenerator.py

```diff
@@ -239,18 +249,28 @@ def qtProjectGenerator(package, argv, extra):
 
     # now go through all checkout dirs to find all header / include files
     for name,path in OrderedDict(sorted(dirs, key=lambda t: t[1])).items():
-        newPath = os.path.join(symlinkDir, name)
-        if not args.update: os.symlink(os.path.join(os.getcwd(), path), newPath)
+        name = name.replace('::', '__')
```

Comment:
Please move the replace in the `isWindows` path. The substitution is not needed on Linux and there we can show the correct package name which looks better.

Reply:
done.